wandb/data_types.py

```diff
@@ -375,7 +418,7 @@ def seq_to_json(self, seq, run, key, step):
 
 
 class Table(Media):
-    """This is a table designed to display small sets of records.
+    """This is a table designed to display sets of records.
```

Comment:
I think we should say "store" instead of "display"

Reply:
incoming